script.module.pysubs2/addon.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
+<addon id="script.module.pysubs2" name="pysubs2 module" version="1.0.0" provider-name="bkiziuk">
```

Comment:
Author is the original author unless this is patched

Reply:
Got it.
Regarding the new version, I missed the latest release by a dozen of hours ;-). Let me correct that.